addons/web/static/src/search/search_model.js

```diff
@@ -1844,11 +1844,14 @@ export class SearchModel extends EventBus {
      * of a search item of type 'field'.
      */
     _getFieldDomain(field, autocompleteValues) {
-        const domains = autocompleteValues.map(({ label, value, operator }) => {
+        const domains = autocompleteValues.map(({ label, value, operator, enforceEqual }) => {
             let domain;
             if (field.filterDomain) {
+                if (enforceEqual) {
+                    field.filterDomain = field.filterDomain.replaceAll("'ilike'", "'='");
```

Comment:
you cannot do that. next time you search on the same field without "s it won't work

Reply:
check if "ilike"